python-lib/dku_kube/metrics_server.py

```diff
@@ -0,0 +1,9 @@
+import os, json, logging
+from .kubectl_command import run_with_timeout
+
+def install_metrics_server(kube_config_path):
+    env = os.environ.copy()
+    env['KUBECONFIG'] = kube_config_path
+    cmd = ['kubectl', 'apply', '-f', 'https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml']
+    logging.info("Installing Metrics Server with : %s" % json.dumps(cmd))
+    out, err = run_with_timeout(cmd, env=env, timeout=30)
```

Comment:
I think it would be a be good idea to put a try/except around this call, since we probably don't want the entire "create cluster" step to fail if the metrics server install fails.

Reply:
👍 Good catch, will do the change.